app/containers/Applications/ApplicationWizardConfirmation.tsx

```diff
@@ -16,22 +18,33 @@ interface Props {
 }
 
 export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Props> = props => {
+  const router = useRouter();
   // Change application status to 'pending' on application submit
   const submitApplication = async () => {
     const {environment} = props.relay;
+    const date = new Date().toUTCString();
+    // TODO: created_at should be set by a trigger
     const variables = {
       input: {
-        id: props.query.application.applicationStatus.id,
-        applicationStatusPatch: {
-          applicationStatus: 'pending'
+        applicationStatus: {
+          applicationId: props.query.application.rowId,
+          applicationStatus: 'PENDING' as CiipApplicationStatus,
+          createdAt: date,
+          createdBy: 'Admin'
         }
       }
     };
-    const response = await updateApplicationStatusMutation(
+    const response = await createApplicationStatusMutation(
       environment,
       variables
     );
     console.log(response);
+    // TODO: check response
+    const newUrl = {
+      pathname: '/complete-submit'
+    };
+    router.push(newUrl, newUrl, {shallow: true});
+    // TODO: do we need shallow?
```

Comment:
we don't need shallow, this can just be `router.push('/complete-submit')`
Shallow routing is for the times where you don't actually change pages, but you want to update the query string, for instance when navigating between application steps